transform/operations.go

```diff
@@ -15,8 +15,8 @@ import (
 
 var durationRe = regexp.MustCompile(`^([\d]*?):?([\d]*):([\d]*)$`)
 
-// duration is a transformOperation which changes from a string duration like "MM:SS" to a number of seconds as
-// an integer.
+// duration is a transformOperation which changes from a string duration like
+// an integer. "MM:SS" to a number of seconds as
```

Comment:
That looks unintentional.

Reply:
You're right, I must placed it back in the wrong order, will fix

Reply:
https://github.com/GannettDigital/jstransform/pull/123/commits/61184f021faeca8a9ec4a0128dddd935b32a25e3